File: test/ayush.dev/scala/pipelines/ScalaInsideProjectDatasetsOnly/code/src/main/scala/io/prophecy/pipelines/scalainsideprojectdatasetsonly/graph/src_parquet.scala
package io.prophecy.pipelines.scalainsideprojectdatasetsonly.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_parquet {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("parquet")
      .load("dbfs:/Prophecy/qa_data/parquet/all_type_no_partition")

}

File: test/mainisnotmain/pipelines/TEST_MAIN/code/test_main/graph/src_parquet.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from test_main.config.ConfigStore import *
from test_main.udfs.UDFs import *

def src_parquet(spark: SparkSession) -> DataFrame:
    return spark.read.format("parquet").load("dbfs:/Prophecy/qa_data/parquet/CustomersDatasetInput.parquet")
